packages/components/tests/dummy/app/templates/components/page-header.hbs

```diff
@@ -0,0 +1,323 @@
+
+      <Shw::Text::Body @tag="p">
+        These examples are meant to showcase the flexiblity, responsiveness, and potential gotchas related to the layout
+        and fluidity of the component.
+      </Shw::Text::Body>
+
+      <Shw::Text::H3>Long title</Shw::Text::H3>
+
+      <Shw::Text::Body @tag="p">Recommended title length is less than four words and not exceed 50 characters.</Shw::Text::Body>
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>Boundary cluster access control and clusters</PH.Title>
+        <PH.IconTile @icon="boundary-color" @color="boundary" />
+        <PH.Actions>
+          <Hds::Dropdown as |DD|>
+            <DD.ToggleButton @text="Manage users" @color="secondary" />
+            <DD.Interactive @text="Assign roles" @icon="user" />
+            <DD.Interactive @text="Batch edit" @icon="edit" />
+            <DD.Interactive @text="Delete user" @icon="trash" />
+          </Hds::Dropdown>
+          <Hds::Button @text="Add user" @icon="plus" @iconPosition="leading" />
+        </PH.Actions>
+      </Hds::PageHeader>
+
+      <Shw::Divider @level={{2}} />
+
+      <Shw::Text::H3>Long title + badge</Shw::Text::H3>
+
+      <Shw::Text::Body @tag="p">
+        In the case of a badge, the title and badge will wrap if the available space is exceeded.
+      </Shw::Text::Body>
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>Long page title with a badge ipsum dolor sit amet</PH.Title>
+        <PH.IconTile @logo="vagrant" />
+        <PH.Badges>
+          <Hds::Badge @text="Wrapping badge" @icon="mic" @color="critical" />
+        </PH.Badges>
+        <PH.Actions>
+          <Hds::Button @text="Get started" @icon="terminal-screen" @iconPosition="trailing" />
+        </PH.Actions>
+      </Hds::PageHeader>
+
+      <Shw::Divider @level={{2}} />
+
+      <Shw::Text::H3>Component wrapping</Shw::Text::H3>
+
+      <Shw::Text::Body @tag="p">The component uses container queries to determine two breakpoints at
+        <code>700px</code>
+        and
+        <code>400px</code>
+        which stack the content in a single column.
+      </Shw::Text::Body>
+
+      <Shw::Grid @columns={{3}} as |SG|>
+
+        <SG.Item>
+          <Hds::PageHeader as |PH|>
+            <PH.Title>Title of the page</PH.Title>
+            <PH.IconTile @icon="smartphone" />
+            <PH.Breadcrumb>
+              <Hds::Breadcrumb>
+                <Hds::Breadcrumb::Item @text="Organization" />
+                <Hds::Breadcrumb::Item @text="User" />
+              </Hds::Breadcrumb>
+            </PH.Breadcrumb>
+            <PH.Badges>
+              <Hds::Badge @text="Small viewport" @icon="tablet" />
+            </PH.Badges>
+            <PH.Subtitle>Forced small viewport</PH.Subtitle>
+            <PH.Actions>
+              <Hds::Button @text="Edit" @color="secondary" />
+              <Hds::Button @text="Create" @color="primary" />
+            </PH.Actions>
+          </Hds::PageHeader>
+        </SG.Item>
+
+      </Shw::Grid>
```

Comment:
Maybe instead of using a grid with three columns, what you could do is to have the same component inside three different containers, with different widths. Something like this:

```suggestion
      <Shw::Flex @direction="column" as |SF|>
        {{#let (array "399px" "767px" "100%") as |widths|}}
          {{#each widths as |width|}}
            <SF.Item @label="container width = {{width}}">
              <Shw::Outliner {{style width=width}}>
                <Hds::PageHeader as |PH|>
                  <PH.Title>Title of the page</PH.Title>
                  <PH.IconTile @icon="smartphone" />
                  <PH.Breadcrumb>
                    <Hds::Breadcrumb>
                      <Hds::Breadcrumb::Item @text="Organization" />
                      <Hds::Breadcrumb::Item @text="User" />
                    </Hds::Breadcrumb>
                  </PH.Breadcrumb>
                  <PH.Badges>
                    <Hds::Badge @text="Small viewport" @icon="tablet" />
                  </PH.Badges>
                  <PH.Subtitle>Forced small viewport</PH.Subtitle>
                  <PH.Actions>
                    <Hds::Button @text="Edit" @color="secondary" />
                    <Hds::Button @text="Create" @color="primary" />
                  </PH.Actions>
                </Hds::PageHeader>
              </Shw::Outliner>
            </SF.Item>
          {{/each}}
        {{/let}}
      </Shw::Flex>
```

Reply:
That's much more elegant!